src/multirust-cli/multirust_mode.rs

```diff
@@ -326,7 +326,7 @@ fn add_target(cfg: &Cfg, m: &ArgMatches) -> Result<()> {
         pkg: "rust-std".to_string(),
         target: TargetTriple::from_str(target),
     };
-    try!(toolchain.add_component(new_component));
+    try!(toolchain.add_component(new_component, &cfg));
```

Comment:
We can't pass `Cfg` to `Toolchain` - it's coupling these two types in the wrong direction. I'll try to offer suggestions how to change it.